src/InternPool.zig

```diff
@@ -1464,7 +1464,7 @@ pub const NamespaceIndex = enum(u32) {
                 unwrapped.index);
         }
     };
-    fn unwrap(namespace_index: NamespaceIndex, ip: *const InternPool) Unwrapped {
+    pub fn unwrap(namespace_index: NamespaceIndex, ip: *const InternPool) Unwrapped {
```

Comment:
Are you using these anywhere? These are unwrapping an *index* to the internal repr of tid+index. AFAICT, you're only using optional index unwrap, which are already `pub`.

These are intentionally private to avoid accidentally unwrapping into internal data outside of InternPool.

Reply:
Nope, removing!